### ISSUE TITLE
When including a (specific) plugin, datepicker throws error on calendar icon click

### ISSUE THREAD

**Issue by kswedberg** [formkit/formkit#1439]
### Reproduction

https://formkit.link/faa7da77f917c1e8bc72dfd3ddd866e9

### Describe the bug

(See attached reproduction link)

Using this plugin with datepicker input:

```js
const todayDatePlugin = (node) => {
    if (node.props.type !== 'datepicker') return;

    const schemaFn = node.props.definition.schema

    node.context.fns.isToday = (day) => {
      return node.context.fns.format(new Date(), 'short') === day;
    }

    node.props.definition.schema = (sectionsSchema = {}) => {
      sectionsSchema.dayCell = {
        attrs: {
          'date-is-today': '$fns.isToday($fns.format($day, "short"))'
        }
      };
      
      return schemaFn(sectionsSchema)
    }
  }
```

The error appears in Tempo (parts.ts), but according to @justin-schroeder that is because the line `return node.context.fns.format(new Date(), 'short') === day;` isn't returning the expected value.

### Environment

• OS: any
• Browser: Chrome
• Version: 1.6.x (latest) in playground / 1.6.5 in personal app

**Comment by kswedberg** [comment]
Thanks, @justin-schroeder ! I'll just wait until you change `$day` back to a `Date` object. For now, i've commented out that line in my code.

**Comment by sashamilenkovic** [comment]
@kswedberg This should be resolved at FormKit Pro v0.126.14

**Comment by kswedberg** [comment]
thanks so much!
